cluster-autoscaler/integration/framework.go

```diff
@@ -176,20 +181,21 @@ func (driver *Driver) runAutoscaler() {
 		return
 	}
 
-	if len(machineDeployments.Items) > 3 {
+	if len(machineDeployments.Items) > 4 {
```

Comment:
I am not sure why 4 MDs are required when you in the test that you have added you only need 1 machine deployment. Can you clarify the reasons for doing so?

Reply:
The IT that I have added needs both workloads to be deployed on nodes in the same machine deployment. I need a worker pool with 1 zone for this to be absolutely certain. If there are three zones, then one workload can go to one zone, and the other one can go to the other zone. In the current setup, we deploy a workload to a worker, not a specific machine deployment.